pom.xml

```diff
@@ -13,60 +13,52 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
+
         <aspectj.version>1.9.5</aspectj.version>
         <allure-maven.version>2.10.0</allure-maven.version>
-        <testng.parallel>classes</testng.parallel>
-        <testng.threadCount>5</testng.threadCount>
+        <jackson.version>2.12.2</jackson.version>
+        <lombok.version>1.18.12</lombok.version>
+        <slf4j.version>1.7.25</slf4j.version>
```

Comment:
What kind of endorsement are you talking about here?

Prefer to stick to using SLF4J since that provides a nice facade for Logging and is more widely used as compared to JUL for [specific reasons](https://stackoverflow.com/questions/11359187/why-not-use-java-util-logging). Using open source libraries IMO are seen as more that you understand the ecosystem as opposed to an endorsement of an individual or an organisation.
May not be relevant but slf4j is distributed under the MIT license which is the most permissive free licenses available.